app/views/users/noticed_notifications/index.html.erb

```diff
@@ -11,11 +11,20 @@
       <span id="all-notifications" class="active notifications-tab" data-action="click->notifications#activeAllNotifications"><%= t("users.notifications.all") %></span>
       <span id="unread-notifications" class="notifications-tab" data-action="click->notifications#activeUnreadNotifications"><%= t("users.notifications.unread") %></span>
     </div>
-    <% if @unread.count.positive? %>
+    <div>
+    <% if @notification.count.positive? %>
       <div>
         <%= link_to t("users.notifications.mark_as_read_button"), mark_all_as_read_url(current_user), class: "anchor-text" %>
       </div>
     <% end %>
+    </div>
+    <div>
+    <% if @notification.count.positive? %>
+      <div>
+        <%= link_to t("users.notifications.delete_all_notifications"), delete_all_notifications_url(current_user), class: "anchor-text" %>
+      </div>
+  <% end %>
```

Comment:
```suggestion
      <% if @unread.count.positive? %>
         <%= link_to t("users.notifications.mark_as_read_button"), mark_all_as_read_url(current_user), class: "anchor-text" %>
      <% end %>
      <% if @notification.count.positive? %>
        <%= link_to "Clear All", delete_all_notifications_url(current_user), class: "anchor-text" %>
      <% end %>
```

- Also "Delete All Notifications" looks very big, make it "Clear All".
- The button should first call for confirmation in the modal. You can refer to how the modal appears each time before asking to remove the "member" from the group.
- The Button color should be `$primary-red`

Reply:
Got it, Thanks for reviewing the PR, I will make the requested changes.